tests/Block/AdminPreviewBlockServiceTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Block;
+
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\FieldDescriptionCollection;
+use Sonata\AdminBundle\Admin\Pool;
+use Sonata\AdminBundle\Block\AdminPreviewBlockService;
+use Sonata\AdminBundle\Datagrid\Datagrid;
+use Sonata\BlockBundle\Test\BlockServiceTestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Twig\Environment;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class AdminPreviewBlockServiceTest extends BlockServiceTestCase
+{
+    /**
+     * @var Pool
+     */
+    private $pool;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->pool = new Pool(new Container());
+    }
+
+    public function testDefaultSettings(): void
+    {
+        $blockService = new AdminPreviewBlockService($this->createMock(Environment::class), $this->pool);
+        $blockContext = $this->getBlockContext($blockService);
+
+        $this->assertSettings([
+            'text' => 'Preview',
+            'filters' => [],
+            'icon' => false,
+            'limit' => 10,
+            'code' => false,
+            'template' => '@SonataAdmin/Block/block_admin_preview.html.twig',
+            'remove_list_fields' => ['_action'],
```

Comment:
I forgot here to mention to change to the constant 😞, sorry

Reply:
Thank you!